Core/GDCore/IDE/Events/EventsRefactorer.cpp

```diff
@@ -631,6 +632,21 @@ vector<EventsSearchResult> EventsRefactorer::SearchInEvents(
       }
     }
 
+    if (inComments) {
+      vector<gd::String> stringsVectors = events[i].GetAllSearchableStrings();
+      
+      for (std::size_t j = 0; j < stringsVectors.size(); ++j) {

```

Comment:
Iterating here is doable, but then you iterate also in SearchStringInComments... you have to choose where! :) 
Remove this loop